src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiEdmTypeSchemaGenerator.cs

```diff
@@ -167,53 +169,54 @@ public static OpenApiSchema CreateSchema(this ODataContext context, IEdmPrimitiv
                     schema.Pattern = "^[0-9]{4,}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]([.][0-9]{1,12})?(Z|[+-][0-9][0-9]:[0-9][0-9])$";
                     break;
                 case EdmPrimitiveTypeKind.Decimal: // decimal
+                    format = "decimal";
```

Comment:
as far as I can tell the format variable is only used in the switch case where it's assigned, maybe we could get rid of the variable and use the object initialization directly for sanity now?

Reply:
The variable used here is to avoid duplication of the format values within those switch statement blocks

Reply:
it then should be a local constant instead of a variable to avoid affectations

Reply:
Okay, I agree.